hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -682,6 +697,24 @@ private void audit(AuditAction action, EventType eventType,
     }
   }
 
+  private void performanceAudit(AuditAction action, Map<String, String> params,
+      PerformanceStringBuilder performance) {
+    AuditMessage msg =
+        buildAuditMessageForPerformance(action, params, performance.build());
+    AUDIT.logPerformance(msg);
+  }
+
+  public AuditMessage buildAuditMessageForPerformance(AuditAction op,
+      Map<String, String> auditMap, String performance) {
```

Comment:
`AuditMessage` takes `PerformanceStringBuilder` directly (after HDDS-9822):

```suggestion
      Map<String, String> auditMap, PerformanceStringBuilder performance) {
```

Reply:
Fixed.